source/django_models.rst

```diff
@@ -22,21 +22,32 @@ In the file ``polls/models.py`` type::
         choice_text = models.CharField(max_length=200)
         votes = models.IntegerField(default=0)
 
-By adding new models we have changed the database schema. We need to run ``syncdb`` again so that 
+By adding new models we have changed the database schema. In other words we have to *migrate database schema* from one to another. We need to run ``makemigrations`` and then ``migrate`` so that 
 new models can appear in the database.
 
 .. warning::
-    After executing the ``syncdb``, you can not add new fields to the model. You can only add new models. There are ways to avoid it, but … it is a totally different story
+    After executing the ``makemigrations`` and ``migrate``, you cannot add new fields to the model. You can only add new models. There are ways to avoid it, but... it is a totally different story.
```

Comment:
With v1.8.2's admin panel w have:
```
OperationalError at /admin/polls/poll/
no such column: polls_poll.<new_column>
```
where `<new_column>` is th one added without `makemigrations`+`migrate`. Even when self-checks are OK:
```System check identified no issues (0 silenced).```